source/distributed/tria.cc

```diff
@@ -1326,6 +941,173 @@ namespace
     // get the weight, increment the pointer, and return the weight
     return *this_object->current_pointer++;
   }
+
+
+
+  // Note: We will no longer need the quadrant pointer in this alias
+  //       if no functionality accesses quadrants' user_data anymore, i.e.
+  //       if we do not support p4est lower than 2.0.
+  template <int dim, int spacedim>
+  using quadrant_cell_relation_t = typename std::tuple<
+    typename dealii::internal::p4est::types<dim>::quadrant *,
+    typename dealii::Triangulation<dim, spacedim>::CellStatus,
+    typename dealii::Triangulation<dim, spacedim>::cell_iterator>;
+
+
+
+  template <int dim, int spacedim>
```

Comment:
Can you add a brief descriptive comment to this function and the following ones?